ares/ms/cartridge/board/pak4.cpp

```diff
@@ -0,0 +1,60 @@
+
+struct Pak4 : Interface {
+  using Interface::Interface;
+  Memory::Readable<n8> rom;
+
+  auto load() -> void override {
+    Interface::load(rom, "program.rom");
+  }
+
+  auto save() -> void override {
+  }
+
+  auto unload() -> void override {
+  }
+
+  auto read(n16 address, n8 data) -> n8 override {
+    if(address <= 0xbfff)
+		return rom.read((romBank[address>>14]<<14) + (address & 0x3fff));
```

Comment:
nit: if statements without braces usually should have their body on the same line 
nit: spaces around operators